content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,118 @@
+
+
+### AdHoc
+AdHoc includes five overlay tests:
+
+- Automated checks (same test within each mode)
+- Landmarks
+- Color
+- Tab stops (same test within FastPass)
+- Headings
+
+![""](https://user-images.githubusercontent.com/40274682/89365128-d4a99000-d688-11ea-9db5-c29cb4a55075.png)
+
+Each section has a link on understanding how to test, dos and don'ts, and best practices. 
+
+![""](https://user-images.githubusercontent.com/40274682/89365012-9f9d3d80-d688-11ea-8f26-bc918a20e34b.png)
+
+### Download:
+For the web/electron: 
+- [Accessibility Insights for Chrome](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
+- [Accessibility Insights for Edge](https://microsoftedge.microsoft.com/addons/detail/ghbhpcookfemncgoinjblecnilppimih)
+
+For mobile apps: 
+- [Accessibility Insights for Android](https://accessibilityinsights.io/en/downloads). 
+
+## Accessibility guide for developers
+An official site of the United States government, their [front-end development](https://accessibility.digital.gov/front-end/getting-started/) section provides a general walkthrough of standards to meet for hitting WCAG 2.0 on government websites. Because it only reaches WCAG 2.0, there are WCAG 2.1 specific guidelines that are missing, but overall it gives a good picture of content categories to think about while developing accessibly. 
+
+## The A11y project
+[The A11y project](https://www.a11yproject.com/#learn-the-fundamentals-and-principles-behind-accessible-design) is a resource containing helpful blog posts, a [manual accessibility checklist](https://www.a11yproject.com/checklist/), and a plethora of other [resources](https://www.a11yproject.com/resources/). The resources list is long, so we recommend navigating via the table of contents, or searching the page with CMD+F for specific queries. 
+
+## a11y portal
+[Another manual a11y checklist](https://www.a11yportal.com/guidelines/fundamentals/checklists.html#custom-control-checklist-developer), specifically for custom control creation.
+
+We recommend familiarizing yourself with the items in this checklist and using them as a resource when authoring new code. As always, the best source of truth for the last accessibility guidelines can be found on the [W3C Web Accessibility Initiative website](https://www.w3.org/WAI/). 
+
+## Searching the internet
+A great way to find answers to very specific accessibility questions when a resource is unavaiable, is to do an online search for the component/issue in question + 'a11y', 'accessible', or 'accessibility', and scan for official websites like W3C ([https://w3.org](https://w3.org)). 
+
+Many of those searches will lead you to the [WAI-ARIA authoring practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/) and [Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/TR/wai-aria-1.2/)
+
+Please double check any findings and ask questions before implementing a solution.
+
+## Semantic markup
```

Comment:
Yeah the developer page isn't on the site any more. 

Anything developer-specific I've left out since this is geared more towards design. 

However I left the doc in because it's still good information that should go somewhere (probably in some engineering resource?). Just didn't want to delete it completely and loose all that info 😅. 

I can always remove this file and just save it in my notes somewhere too.